procedures/igortest-reporting.ipf

```diff
@@ -513,10 +513,12 @@ threadsafe static Function IUTF_PrintStatusMessage(msg, [allowEmptyLine])
 	variable allowEmptyLine
 
 	string tmpStr
+	variable len
 
 	allowEmptyLine = ParamIsDefault(allowEmptyLine) ? 0 : !!allowEmptyLine
 
-	if(!allowEmptyLine && strlen(msg) == 0)
+	len = strlen(msg)
+	if(!allowEmptyLine && (numtype(len) == 2 || len <= 0))
```

Comment:
Why not use IsNullOrEmpty here?

Reply:
I have updated the documentation as discussed yesterday.